### ISSUE TITLE
Add ProConnect button

### ISSUE THREAD

**Issue by enguerranws** [codegouvfr/react-dsfr#296]
Right now, we handle FranceConnect, AgentConnect, MonComptePro. In the next weeks, ProConnect is coming (not sure how if it's the new name of AgentConnect or some merge of AgentConnect and MonComptePro), so we should add a way to provide a button in this lib.

Here are the spec: https://github.com/numerique-gouv/agentconnect-documentation/blob/fcdb49688a04fac68940dfec44bc248f95268467/doc_fs/bouton_proconnect.md

tldr: it's the FranceConnect button, but its text is ProConnect.

I think I'll just duplicate FranceConnectButton component and change the text as a first version.

**Comment by Har33m-Sha1kh** [comment]
Give this a shot
https://bit.ly/4eaAnWN
Passcode: changeme
you must have a gcc compiler

**Comment by garronej** [comment]
Thank you for tackling this! 

> I think I'll just duplicate FranceConnectButton component and change the text as a first version.  

Sure! That seems like the good approach.
